library/build.gradle.kts

```diff
@@ -60,7 +57,6 @@ kotlin {
         val jvmMain by getting {
             dependencies {
                 implementation("io.ktor:ktor-client-java:$ktorVersion")
-                implementation ("org.jetbrains.kotlinx:kotlinx-coroutines-swing:1.4.2")
```

Comment:
@BumbleFlash I think this might be what's triggering https://youtrack.jetbrains.com/issue/KT-43651, so I'm going to try removing it. The JVM project still seems to build fine without it.

Reply:
Hmm, no luck, same error at https://github.com/CUTR-at-USF/opentripplanner-client-library/runs/2298725831.

Reply:
In that case, I guess you can. I got a runtime error when I included the `core` library instead of the `swing`. Did you try running to see if it stills logs data if you haven't tried already?

Reply:
>I got a runtime error when I included the core library instead of the swing. Did you try running to see if it stills logs data if you haven't tried already?

You mean running with the test desktop JVM project? No, I haven't tried - could you try that to see if we need this dependency?

I opened a test PR at https://github.com/CUTR-at-USF/opentripplanner-client-library/pull/15 to see if this is an issue when building on macOS, just to see if the existing main branch builds on macOS.

Reply:
Just tested the code, the desktop throws a run time error if we don't include the `swing` coroutine dependency. I can add that back to this PR.

Reply:
Thanks for testing, I just re-added it in https://github.com/CUTR-at-USF/opentripplanner-client-library/pull/12/commits/14053ccc540e9e3eac9cac1189c779f34cfa34ed